provisioning/service/samples/readme.md

```diff
@@ -0,0 +1,72 @@
+# Samples for the Azure IoT Device Provisioning Service SDK for Node.js
+
+This file can be found in http://github.com/azure/azure-iot-sdk-node/provisioning/service/samples
+
+This folder contains simple samples showing how to use the various features of the Microsoft Azure IoT Device Provisioning Service from an application running JavaScript code.
+
+## List of samples
+
+* Create a simple IndividualEnrollment object.
+   *  **simple_IndividualEnrollment_create.js**
+* Create a simple EnrollmentGroup object.
+   *  **simple_EnrollmentGroup_create.js**
+* Create a couple of IndividualEnrollments and delete them.
+   *  **createdelete_sample.js**
+* Create queries for IndividualEnrollments, EnrollmentGroups and DeviceRegistrationStates that belong to an EnrollmentGroup.
+   *  **query_sample.js**
+* Perform BulkEnrollmentOperations to do various CRUD operations.
+   *  **bulkupdate_sample.js**
+
+
+## How to run the samples
+
+In order to run the device samples you will first need the following prerequisites:
+* Node.js v0.10 or above on your target device. (Check out [Nodejs.org](https://nodejs.org/) for more info)
+* [Setup Azure IoT Device Provisioning ][lnk-setup-iot-provisioning] Stop before executing the the Create and provision a simulated device section.  The samples below will perform these steps.
+
+<<<<<<<<<<<<<<<<<<link to how to obtain an endorsment key!!!!>>>>>>>>>>>>>>>>>>
+
+Once you have a device identity for your sample, get the following files from the current folder:
+* **package.json**
+* **__sample_file.js__** (where **__sample_file.js__** is one of the files listed above and available in this folder)
+
+Place the files in the folder of your choice on the target machine/device then go through the following steps:
+
+* From a shell or Node.js command prompt, navigate to the folder where you placed the sample files. For creating an IndividualEnrollment do:
+
+```
+$ npm install
+$ node sample_IndividualEnrollment.js "<the connection string for the Device Provisioning instance enclosed in quotes>" "<an endorsement key in quotes>"
```

Comment:
>sample_IndividualEnrollment [](start = 7, length = 27)

missing _create ?

Reply:
also, weird casing mix in the file name is weird

---
In reply to: [150136510](https://github.com/Azure/azure-iot-sdk-node/pull/142#discussion_r150136510) [](ancestors = 150136510)

Reply:
(also if you decide not to change the name, i think it should be "simple" not "sample") but really this prefix doesn't add much value, does it?

---
In reply to: [150136587](https://github.com/Azure/azure-iot-sdk-node/pull/142#discussion_r150136587) [](ancestors = 150136587,150136510)

Reply:
See, the updated file names.

---
In reply to: [150136695](https://github.com/Azure/azure-iot-sdk-node/pull/142#discussion_r150136695) [](ancestors = 150136695,150136587,150136510)